ingestion-beam/src/main/java/com/mozilla/telemetry/contextualservices/SponsoredInteraction.java

```diff
@@ -0,0 +1,119 @@
+package com.mozilla.telemetry.contextualservices;
+
+import com.google.auto.value.AutoValue;
+import com.mozilla.telemetry.ingestion.core.schema.SchemaNotFoundException;
```

Comment:
We should use a different exception type here. If Beam has a built-in exception for this, we should consider that.